stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/view/MessageListView.kt

```diff
@@ -662,7 +662,7 @@ public class MessageListView : ConstraintLayout {
         buffer.enqueueData(listItem)
     }
 
-    public fun setMessageListItemPredicate(messageListItemPredicate: MessageListItemPredicate) {
```

Comment:
Why do you call it filter? It's not filter anymore. Just object that holds predicated

Reply:
We haven't published our docs yet. We can change namings

Reply:
The user is setting the predicate that will be used to filter the messages in the `handleNewWrapper`. For me, filter seams an easier word than predicate. If you think this is misleading, we can maintain predicate.

Reply:
Let's keep the predicate then, we can revisit this in a moment.